mps_youtube/commands/config.py

```diff
@@ -6,9 +6,24 @@
 def showconfig():
     """ Dump config data. """
     width = util.getxy().width
-    width -= 30
-    s = "  %s%-17s%s : %s\n"
-    out = "  %s%-17s   %s%s%s\n" % (c.ul, "Key", "Value", " " * width, c.w)
+    longest_key = 17
+    longest_val = 0
+    has_temps = False
+
+    for setting in config:
+        val = config[setting]
+        longest_val = max(longest_val, len(str(val.display)))
+        has_temps = has_temps or val.temp_value is not None
+
+    width -= 27
+    s = f"  %s%-{longest_key}s%s : %-{longest_val+1}s"
+
+    if has_temps:
+        width -= longest_val + 5
+        out = (f"  %s%-{longest_key}s   %-{longest_val}s %s%s%s\n") % (
```

Comment:
@BlackCapCoder You seemed to introduce python 3.6+ f-strings to the repo. Even though I love this feature, not everyone is on python 3.6 as of now.

Can you send in a quick PR to change this and use `.format` instead?

If you're busy, then that's fine.